vm_operator/charmcraft.yaml

```diff
@@ -0,0 +1,17 @@
+# This file configures Charmcraft.
+# See https://juju.is/docs/sdk/charmcraft-config for guidance.
+
+type: charm
+bases:
+  - build-on:
+    - name: ubuntu
+      channel: "22.04"
+    run-on:
+    - name: ubuntu
+      channel: "22.04"
+
+parts:
+  charm:
+    charm-binary-python-packages:
+      - psycopg[binary]
+      - PyYAML
```

Comment:
I *think* you can remove `PyYAML` from here now; this was due to a bug in the upstream that stopped the wheel building from source, which is now fixed.